terraform/github/main.tf

```
@@ -293,7 +305,8 @@ module "core-team" {
     module.modernisation-platform-cp-network-test.repository.name,
     module.modernisation-platform-terraform-module-template.repository.name,
     module.modernisation-platform-terraform-pagerduty-integration.repository.name,
-    module.terraform-module-github-oidc-provider.repository.name
+    module.terraform-module-github-oidc-provider.repository.name,
+    module.modernisation-platform-configuration-management
```

```suggestion
    module.modernisation-platform-configuration-management.repository.name
```
The plan is failing with the name attribute missing

---

updated